src/routers/genericRouters/routerEdges.ts

```diff
@@ -82,6 +82,31 @@ export class RouterEdges extends RouterFilterBy {
     return await cursor.all()
   }
 
+  // A -(edge)-> B, given ID for A and B, return A or B (opposite of ID match), and edge
+  async getCompleteBidirectionalByID (input: paramsFormatType, idName: string, page: number = 0, sortBy: string = ''): Promise<any[]> {
+    const recordId = input[idName] as string
+
+    // eslint-disable-next-line @typescript-eslint/no-dynamic-delete
+    delete input[idName]
+
+    const id = `${this.sourceSchemaCollection}/${decodeURIComponent(recordId)}`
+
+    let filters = this.getFilterStatements(input)
+    if (filters) {
+      filters = ` AND ${filters}`
+    }
+
+    const query = `
+        FOR record IN ${this.edgeCollection}
+          FILTER (record._from == '${id}' OR record._to == '${id}') ${filters}
+          ${this.sortByStatement(sortBy)}
+          LIMIT ${page * QUERY_LIMIT}, ${QUERY_LIMIT}
+          RETURN {[record._from == '${id}' ? '${this.sourceSchemaName}' : '${this.targetSchemaName}']: UNSET(DOCUMENT(record._from == '${id}' ? record._to : record._from), '_rev', '_id'), ${this.dbReturnStatements}}
```

Comment:
Can you add an example for return here? It is hard to image what the return looks like. Thank you!

Reply:
Thanks a lot!